src/06-loops.js

```diff
@@ -61,7 +84,16 @@ function rangeEveryOther(min, max) {}
  *  containsWhileLoop([ "left", "up", "right" ], "down");
  *  //> false
  */
-function containsWhileLoop(array, target) {}
+function containsWhileLoop(array, target) {
+  let i = 0;
+
+  while (i < array.length) {
+    i++;
+    if (array.indexOf(target) !== -1) {
+      return true;
+    } else return false;
+  }
+}
```

Comment:
You pass the test (solve the problem with a while loop), but `.indexOf` is actually doing all the work. Your function could be simply `return array.indexOf(target) !== -1` and you'd return the right result! Your `while` loop is jut for show here, but see if you can USE it to solve the problem WITHOUT `.indexOf` or `.includes`.

Reply:
And the same for the `for` loop contains problem below.